plugins/inventory/proxmox.py

```diff
@@ -222,15 +244,24 @@ def _get_session(self):
     def _get_auth(self):
         credentials = urlencode({'username': self.proxmox_user, 'password': self.proxmox_password, })
 
-        a = self._get_session()
-        ret = a.post('%s/api2/json/access/ticket' % self.proxmox_url, data=credentials)
+        if (self.proxmox_password): 
```

Comment:
This is Python, not C :-)
```suggestion
        if self.proxmox_password:
```